generator/internal/descriptor_utils.cc

```diff
@@ -385,6 +385,8 @@ ParameterCommentSubstitution substitutions[] = {
     {R"""(`{cluster} = '-'`)""", R"""(``{cluster} = '-'``)"""},
     {R"""(`projects/<Project ID or '-'>`)""",
      R"""(``projects/<Project ID or '-'>``)"""},
+    {R"""(`projects/{project}/locations/{location}'`)""",
+     R"""(`projects/{project}/locations/{location}`)"""},
```

Comment:
Nit:  It doesn't look like the substitution corresponds to the comment for the block.

Reply:
Fixed.